ui/nuxeo-document-comments/nuxeo-document-comment.js

```diff
@@ -133,10 +134,10 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
           <h2>[[i18n('comments.deletion.dialog.heading')]]</h2>
           <div>[[_computeConfirmationLabel(comment.numberOfReplies)]]</div>
           <div class="buttons">
-            <paper-button name="dismiss" dialog-dismiss
+            <paper-button name="dismiss" dialog-dismiss class="secondary small"
```

Comment:
What is the criteria to choose for small buttons here and not in other dialogs?

Reply:
good catch, all dialogs should be consistent, removing small class from here.